tools/walletextension/main/cli.go

```diff
@@ -59,6 +59,18 @@ const (
 	storeIncomingTxs        = "storeIncomingTxs"
 	storeIncomingTxsDefault = true
 	storeIncomingTxsUsage   = "Flag to enable storing incoming transactions in the database for debugging purposes. Default: true"
+
+	rateLimitThresholdName    = "rateLimitThreshold"
+	rateLimitThresholdDefault = 1000
+	rateLimitThresholdUsage   = "Rate limit threshold per user. Default: 1000."
+
+	rateLimitDecayName    = "rateLimitDecay"
+	rateLimitDecayDefault = 0.2
```

Comment:
If we make this "1", then we keep parity with the milliseconds, and we can reason in units of time

Reply:
fixed

Reply:
yeah, but it means that we allow by default for every user to use 1000ms in 1000ms on average (100%). Rate limits will be hit very rarely and only with concurrent requests.